app/views/admins/_pagination.html.erb

```diff
@@ -0,0 +1 @@
+<%== pagy_bootstrap_nav(@pagy) if @pagy.pages > 1 %>
```

Comment:
``<%==``

Reply:
pagy gem requires the use of <%== %> instead of <%= %>